x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -540,6 +609,21 @@ export const RulesList: React.FunctionComponent = () => {
             ))}
           </EuiFilterGroup>
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButton
```

Comment:
just curious, are we adding this refresh button to the stack management rules list as well?

Reply:
Yep we decided to add it to the Stack Management as well.